### ISSUE TITLE
Is there a way to localize this addon?

### ISSUE THREAD

**Issue by klint** [Rob--W/bookmark-container-tab#7]
I'm ready to help in French.
Thanks

**Comment by Rob--W** [comment]
Yes, it is! Locally I already made the extension localizable. I haven't pushed the changes yet, because I wanted a way to verify it first, and for that I would depend on https://github.com/mozilla/web-ext/issues/1313

**Comment by klint** [comment]
Ok, so let me know when I can help :)
Thanks

**Comment by Rob--W** [comment]
I just published an update with one translation (my native language, Dutch (nl)).

I didn't want to wait until I found time for adding localization support to `web-ext`.
